package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hubii-core",
-  "version": "1.3.0",
+  "version": "1.4.0-beta.1",
```

Comment:
In my view we should take this out as a production release and hence drop the pre-release tag.